src/app/components/team/Statuses/TranslateStatuses.js

```diff
@@ -0,0 +1,164 @@
+      }
+
+      return status;
+    });
+
+    setShowWarning(false);
+
+    if (onSubmit) {
+      onSubmit(newStatusesArray);
+    }
+  };
+
+  const handleSave = () => {
+    let missingTranslation = false;
+
+    statuses.forEach((s) => {
+      const savedTranslation =
+        s.locales[currentLanguage] &&
+        s.locales[currentLanguage].label;
+      const newTranslation = translations[s.id];
+      if (!savedTranslation && !newTranslation) {
+        missingTranslation = true;
+      }
+    });
+
+    if (missingTranslation) {
+      setShowWarning(missingTranslation);
+    } else {
+      handleSubmit();
+    }
+  };
+
+  const handleTextChange = (id, text) => {
+    const newTranslations = { ...translations };
+    newTranslations[id] = text;
+    setStranslations(newTranslations);
+  };
+
+  return (
+    <StyledTranslateStatusesContainer>
+      <Grid container spacing={2}>
+        <Grid item xs={6}>
+          <Typography variant="button">
+            {languagesList[defaultLanguage].nativeName}
+          </Typography>
+        </Grid>
+        <Grid item xs={6}>
+          <StyledColHeader>
+            <Typography variant="button">
+              {languagesList[currentLanguage].nativeName}
+            </Typography>
+            <Button className="translate-statuses__save" variant="contained" color="primary" onClick={handleSave}>
+              <FormattedMessage
+                {...globalStrings.save}
+              />
+            </Button>
+          </StyledColHeader>
+        </Grid>
+      </Grid>
+      {statuses.map(s => (
+        <Grid
+          spacing={2}
+          container
+          direction="row"
+          alignItems="center"
+          key={s.id}
+        >
+          <Grid item xs={6}>
+            <StyledStatusLabel color={s.style.color}>
+              {s.label}
+            </StyledStatusLabel>
+          </Grid>
+          <Grid item xs={6}>
+            <TextField
+              className="translate-statuses__input"
+              defaultValue={
+                s.locales[currentLanguage] ?
+                  s.locales[currentLanguage].label : ''
+              }
+              fullWidth
+              id={`translate-statuses__input-${s.id}`}
+              onChange={e => (handleTextChange(s.id, e.target.value))}
+              size="small"
+              variant="outlined"
+            />
+          </Grid>
+        </Grid>
+      ))}
+      <ConfirmProceedDialog
+        open={showWarning}
+        title={
+          <FormattedMessage
+            id="translateStatuses.missingTranslations"
+            defaultMessage="Missing translations"
+          />
+        }
+        body={
+          <FormattedMessage
+            id="translateStatuses.missingTranslationsBody"
+            defaultMessage="Some statuses are missing translation. If they need to be used in this language, they will be replaced by the same value in the default language, and may not be legible to users."
```

Comment:
Agreement error in the first sentence. Also, does it run a bit long? How about:

> Some statuses are missing translations. Users may not be able to read untranslated statuses.

Reply:
Agree. Please use the copy suggested by @adamhooper. Apologize for not providing the copy for this modale @amoedoamorim. Please don't hesitate to reach out (Figma/Slack prob the best at the design stage) if scenarios are missing from Figma. .